### PR TITLE
add gh-extension topic

### PR DESCRIPTION
**Pull request by vilmibm**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [ ] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

Hi! I'm on the GitHub CLI team and we're using the `gh-extension` topic to organize extensions to the GitHub CLI which we support as of 2.0. Figured it would be nice to have the topic be curated <3 thank you!

### CODE REVIEW

File: topics/gh-extension/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: GitHub
+display_name: gh extension
+github_url: https://github.com/cli/cli
+logo: gh.png
```
Comment (kenyonj) [inline topics/gh-extension/index.md:5]:
```suggestion
logo: gh-extension.png
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
You'll need to update the image to be 288x288, and <75kb. Here is an example if you want to use it:

gh-extension

Also, it'll have to be named `gh-extension.png` as CI is expecting it to be the same name as the topic.

**Review by jasawan — APPROVED** [review]

**Review by jn23-c — APPROVED** [review]

**Comment by Arihayand** [comment]
Nuber

2021-09-12 4:11 GMT-07:00, jn23-c ***@***.***>:

**Comment by vilmibm** [comment]
@kenyonj thanks for the help and sorry for the delayed response ^_^() updated!

**Comment by smashwilson** [comment]
The image looks good now, but it looks like we have two more failures in CI to address:

```
  1) Failure:
topics::gh-extension topic#test_0004_uses the right format for 'released' [/home/runner/work/explore/explore/test/topics_test.rb:52]:
please format 'released' like MONTH DD, YYYY with the month in English.
Expected ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"] to include "Septemberth".

  2) Failure:
topics::gh-extension topic#test_0006_ends 'short_description' with punctuation [/home/runner/work/explore/explore/test/topics_test.rb:78]:
short_description should end with punctuation.
Expected /[.?!]\z/ to match "Extensions to the GitHub CLI".
```

File: topics/gh-extension/index.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: GitHub
+display_name: gh extension
+github_url: https://github.com/cli/cli
+logo: gh-extension.png
+released: September 16th, 2020
+short_description: Extensions to the GitHub CLI
```
Comment (mislav) [inline topics/gh-extension/index.md:7]: How about:
> Community extensions for the GitHub CLI.

or:
> Third-party extensions for the GitHub CLI.

File: topics/gh-extension/index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+topic: gh-extension
+url: https://cli.github.com
+---
+gh is GitHub's official CLI client and it can be extended by anyone through its extension system. Repositories with this topic are able to be installed, giving gh new behaviors.
```
Comment (mislav) [inline topics/gh-extension/index.md:11]: I can imagine that this description could be simpler; something like:

> GitHub CLI extensions are community-maintained repositories that, when installed locally, add extra functionality to gh commands.

File: topics/gh-extension/index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+=
```
Comment (mislav) [inline topics/gh-extension/index.md:12]: Is the `=` special syntax for something?

**Review by mislav — APPROVED** [review]
Some wording suggestions

**Review by dawgwe1 — CHANGES_REQUESTED** [review]
Close all open projects I don’t have any business Organization and I don’t use GitHub

File: topics/gh-extension/index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+created_by: GitHub
+display_name: gh extension
```
[the remark this answers is not in the document]
Comment (kenyonj) [reply topics/gh-extension/index.md:3]:
```suggestion
display_name: GitHub CLI extension
```

File: topics/gh-extension/index.md (hunk 2 above)
Comment (kenyonj) [reply topics/gh-extension/index.md:7]:
```suggestion
short_description: Community extensions for the GitHub CLI.
```

File: topics/gh-extension/index.md (hunk 3 above)
Comment (kenyonj) [reply topics/gh-extension/index.md:11]:
```suggestion
GitHub CLI extensions are community-maintained repositories that, when installed locally, add extra functionality to `gh` commands.
```

File: topics/gh-extension/index.md (hunk 4 above)
Comment (kenyonj) [reply topics/gh-extension/index.md:12]: I'm not sure what it does either, even though I've seen it before in markdown. Maybe a horizontal rule? Let's remove it.
```suggestion
```

File: topics/gh-extension/index.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+created_by: GitHub
+display_name: GitHub CLI extension
+github_url: https://github.com/cli/cli
+logo: gh-extension.png
+released: September 16th, 2020
```
Comment (kenyonj) [inline topics/gh-extension/index.md:6]:
```suggestion
released: September 16, 2020
```

**Review by kenyonj — APPROVED** [review]

**Pull request merged**